systemv/cupstestppd.c

```diff
@@ -3343,10 +3343,10 @@ check_sizes(ppd_file_t *ppd,		/* I - PPD file */
       }
       else
       {
-        width_tmp  = (fabs(size->width - ceil(size->width)) < 0.1) ?
-	                 ceil(size->width) : size->width;
-        length_tmp = (fabs(size->length - ceil(size->length)) < 0.1) ?
-	                 ceil(size->length) : size->length;
+        width_tmp  = (fabs(size->width - ceilf(size->width)) < 0.1f) ?
```

Comment:
ceilf is not portable.

Reply:
ceilf has been part of the standard since c99.

Reply:
Wait, is this supposed to be ANSI C?